tests/features/data-visibility-and-review.feature

```diff
@@ -16,78 +16,107 @@ Feature: Manage data visibility separately from review and approval in the backe
 
   Background:
     Given a functioning api
-    And the dataset model has a visibility field that supports "internal" and "public"
-    And the dataset model has a review_status field that supports "provisional" and "approved"
-    And visibility and review_status are required fields when creating new datasets
-    And new datasets must use visibility and review_status as the source of truth
+    And all database models have a visibility field that supports "internal" and "public"
+    And all database models have a review_status field that supports "provisional" and "approved"
+    And visibility and review_status are required fields when creating new records
+    # hard to test?
+    And new records must use visibility and review_status as the source of truth
     And legacy combined visibility/review fields are deprecated
 
-  Scenario: Require visibility and review status when creating a new dataset
-    When the user attempts to create a new dataset via the api without specifying visibility or review_status
+
+  Scenario Outline: Require visibility and review status when creating a new record

```

Comment:
should these be required fields in the schemas? If so, the response will be 422 from Pydantic